nss-evening-cohort-13/jurassic-world-visitor-dashboard-jurassic-world-visitor-dashboard
# Milestone 2 (Create) done!

## Description
- all the modules have a create button that launches a form and adds the data to firebase

## Related Issue
n/a

## Motivation and Context
we need to be able to add data before we do anything with it

## How Can This Be Tested?
- go to our [deployed site](https://jurassic-world-eb567.web.app/)
- try entering things into each form 
- every form should capture the data and send to our firebase (you should have gotten an email giving you access)

## Screenshots (if appropriate):

## Types of changes
